src/common/config/src/wal/kafka.rs

```diff
@@ -40,34 +49,86 @@ pub struct KafkaConfig {
     pub linger: Duration,
     /// The maximum amount of time (in milliseconds) to wait for Kafka records to be returned.
     #[serde(with = "humantime_serde")]
-    pub max_wait_time: Duration,
+    pub produce_record_timeout: Duration,
+    /// The backoff config.
+    #[serde(flatten, with = "kafka_backoff")]
+    pub backoff: KafkaBackoffConfig,
+}
+
+impl Default for KafkaConfig {
+    fn default() -> Self {
+        Self {
+            broker_endpoints: vec!["127.0.0.1:9090".to_string()],
```

Comment:
```suggestion
            broker_endpoints: vec!["127.0.0.1:9092".to_string()],
```

The exposed broker endpoint by a Kafka cluster spawned by a docker-compose file is `127.0.0.1:9092` by default. Please change all usages to 9092.